### PR TITLE
Sửa 1 vài lỗi về giao diện

### PR DESCRIPTION
**Pull request by nguyentien98**
https://edu-redmine.framgia.vn/issues/2952

### CODE REVIEW

File: app/Repositories/LocationRepository.php [hunk 1]
```diff
@@ -140,7 +140,8 @@ public function analystic(
                         'id' => $location->id,
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Whitespace found at end of line

File: app/Repositories/LocationRepository.php [hunk 2]
```diff
@@ -140,7 +140,8 @@ public function analystic(
                         'id' => $location->id, 
                         'date' => $date,
                     ]
-                ) . '?shift=' . $filter['shift']
+                ) . '?shift=' . $filter['shift'],
+                'description' => __('Click to view detail')
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Each line in an array declaration must end in a comma

File: app/Repositories/LocationRepository.php [hunk 3]
```diff
@@ -137,10 +137,11 @@ public function analystic(
                 'url' => route(
                     'calendar.location.detail_location',
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Whitespace found at end of line

File: app/Repositories/LocationRepository.php [hunk 4]
```diff
@@ -173,9 +174,9 @@ public function getLocationDetail($filter)
             ->where('date', $filter['date']);
         if (array_key_exists('shift', $filter) && $filter['shift'] != 0) {
             $listUserId = $listUserId->whereIn(
-                'shift', 
+                'shift',
                 [
-                    $filter['shift'], 
+                    $filter['shift'],
                     config('site.shift.all')
                 ]
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php:183]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Comma required after last value in array declaration

File: app/Http/Controllers/UserController.php [hunk 5]
```diff
@@ -29,7 +28,12 @@ class UserController extends Controller
     protected $positionRepository;
     protected $workspaceRepository;
 
-    public function __construct(UserRepository $userRepository, ProgramRepository $programRepository, PositionRepository $positionRepository, WorkspaceRepository $workspaceRepository)
+    public function __construct(
+        UserRepository $userRepository,
+        ProgramRepository $programRepository,
+        PositionRepository $positionRepository,
+        WorkspaceRepository $workspaceRepository
+    )
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Admin/UserController.php [hunk 6]
```diff
@@ -23,13 +23,18 @@ class UserController extends Controller
...
     protected $workspaceRepository;
 
-    public function __construct(UserRepository $userRepository, ProgramRepository $programRepository, PositionRepository $positionRepository, WorkspaceRepository $workspaceRepository)
+    public function __construct(
+        UserRepository $userRepository,
+        ProgramRepository $programRepository,
+        PositionRepository $positionRepository,
+        WorkspaceRepository $workspaceRepository
+    )
     {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserController.php</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Admin/WorkingScheduleController.php [hunk 7]
```diff
@@ -65,44 +65,44 @@ public function chooseWorkplace()
...
             $request,
             [
                 'start' => 'required',
-                'end' => 'required'
+                'end' => 'required',
             ]
         );
         $dates = [
             'start' => $request->start,
             'end' => $request->end
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/WorkingScheduleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/WorkingScheduleController.php</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/Admin/UserDisableController.php [hunk 8]
```diff
@@ -27,13 +27,13 @@ public function __construct(
...
-        WorkspaceRepository $workspaceRepository)
-    {
+        WorkspaceRepository $workspaceRepository
+    ) {
         $this->userRepository = $userRepository;
         $this->programRepository = $programRepository;
         $this->positionRepository = $positionRepository;
         $this->workspaceRepository = $workspaceRepository;
-        $this->middleware('checkTrainer')->except('index', 'create', 'store' , 'edit' , 'update');
+        $this->middleware('checkTrainer')->except('index', 'create', 'store', 'edit' , 'update');
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/UserDisableController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.SpaceBeforeComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserDisableController.php</br>Space found before comma in function call

File: app/Http/Controllers/Admin/UserDisableController.php [hunk 9]
```diff
@@ -72,69 +68,4 @@ public function index(Request $request)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/UserDisableController.php:67]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserDisableController.php</br>The closing brace for the class must go on the next line after the body

File: app/Http/Controllers/WorkScheduleController.php [hunk 10]
```diff
@@ -8,59 +8,59 @@
 use App\Repositories\WorkingScheduleRepository;
 use App\Repositories\LocationRepository;
 
-
 class WorkScheduleController extends Controller
 {
-    public function __construct(WorkingScheduleRepository $workingScheduleRepository, LocationRepository $locationRepository)
-    {
+    public function __construct(
+        WorkingScheduleRepository $workingScheduleRepository,
```
Comment (framgiaciv3) [inline app/Http/Controllers/WorkScheduleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/WorkScheduleController.php</br>Whitespace found at end of line

File: app/Http/Controllers/WorkScheduleController.php [hunk 11]
```diff
@@ -8,59 +8,59 @@
...
+    ) {
         $this->workingSchedule = $workingScheduleRepository;
         $this->locationRepository = $locationRepository;
     }
 
     public function index()
     {
         $dates = [];
-        for ($i = 0 ; $i < 31; $i++) { 
+        for ($i = 0; $i < 31; $i++) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/WorkScheduleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/WorkScheduleController.php</br>Whitespace found at end of line

File: app/Http/Controllers/WorkScheduleController.php [hunk 12]
```diff
@@ -8,59 +8,59 @@
...
             }
         }
 
         $locations = $this->locationRepository->getByWorkspace(Auth::user()->workspace_id);
         $data = $this->workingSchedule->getUserSchedule(Auth::user()->id);
-        $data_location = $this->workingSchedule->getLocation(Auth::user()->id);
+        $dataLocation = $this->workingSchedule->getLocation(Auth::user()->id);
 
-        return view('users.registerworkschedules', compact('dates', 'data', 'locations', 'data_location') );
+        return view('users.registerworkschedules', compact('dates', 'data', 'locations', 'dataLocation') );
```
Comment (framgiaciv3) [inline app/Http/Controllers/WorkScheduleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Controllers/WorkScheduleController.php</br>Expected 0 spaces before closing bracket; 1 found

File: app/Http/Middleware/CheckUser.php [hunk 13]
```diff
@@ -17,18 +17,16 @@ class checkUser
      */
     public function handle($request, Closure $next)
     {
-        if(Auth::check())
-        {   
-            $edit_users = User::findOrFail( $request->route()->parameters('id') );
+        if(Auth::check()) {
```
Comment (framgiaciv3) [inline app/Http/Middleware/CheckUser.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Middleware/CheckUser.php</br>Expected 1 space(s) after IF keyword; 0 found

Comment (framgiaciv3) [inline app/Http/Middleware/CheckUser.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Middleware/CheckUser.php</br>Whitespace found at end of line

File: app/Http/Middleware/CheckAdmin.php [hunk 14]
```diff
@@ -16,8 +16,10 @@ class CheckAdmin
      */
     public function handle($request, Closure $next)
     {
-        if (Auth::user()->role == config('site.permission.admin') || Auth::user()->role == config('site.permission.trainer'))
-        {
+        if (
```
Comment (framgiaciv3) [inline app/Http/Middleware/CheckAdmin.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.ControlStructureSpacing.SpacingAfterOpenBrace</br>**Severity**: error</br>**File**: app/Http/Middleware/CheckAdmin.php</br>Expected 0 spaces after opening bracket; newline found

File: app/Http/Middleware/CheckAdmin.php [hunk 15]
```diff
@@ -16,8 +16,10 @@ class CheckAdmin
...
+            Auth::user()->role == config('site.permission.admin')
```
Comment (framgiaciv3) [inline app/Http/Middleware/CheckAdmin.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Middleware/CheckAdmin.php</br>Whitespace found at end of line

File: app/Repositories/LocationRepository.php [hunk 16]
```diff
@@ -35,7 +35,15 @@ public function listLocation($filter)
 
     public function getByWorkspace($workspaceId)
     {
-        return $this->model->where('workspace_id', $workspaceId)->pluck('name', 'id')->prepend(__('--Chose Location--'), config('site.default_location'))->toArray();
+        return $this->model
+            ->where('workspace_id', $workspaceId)
+            ->pluck('name', 'id')
+<<<<<<< Updated upstream
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Line indented incorrectly; expected at least 8 spaces, found 0

File: app/Repositories/LocationRepository.php [hunk 17]
```diff
@@ -35,7 +35,15 @@ public function listLocation($filter)
...
+            ->prepend(__('--Chose Location--'), config('site.default_location'))->toArray();
+=======
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Line indented incorrectly; expected at least 8 spaces, found 0

File: app/Repositories/LocationRepository.php [hunk 18]
```diff
@@ -35,7 +35,15 @@ public function listLocation($filter)
...
+            ->prepend(__('--Chose Location--'), config('site.default_location'))
+            ->toArray();
+>>>>>>> Stashed changes
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Line indented incorrectly; expected at least 8 spaces, found 0

File: app/Repositories/ProgramRepository.php [hunk 19]
```diff
@@ -1,6 +1,7 @@
 <?php
```
Comment (framgiaciv3) [inline app/Repositories/ProgramRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/ProgramRepository.php</br>Whitespace found at end of line

File: app/Models/User.php [hunk 20]
```diff
@@ -78,7 +78,7 @@ public function scopeFilterUser($query, QueryFilter $filters)
 
     public function getShiftByDate($date)
     {
-        $work_schedule = $this->work_schedules()
+        $work_schedule = $this->workSchedules()
```
Comment (framgiaciv3) [inline app/Models/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Models/User.php</br>Variable "work_schedule" is not in valid camel caps format

File: app/Http/Controllers/Admin/UserDisableController.php (hunk 9 above)
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/UserDisableController.php:67]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserDisableController.php</br>The closing brace for the class must go on the next line after the body

**Comment by dangminhtruong** [comment]
Vẫn còn lỗi convetion vài chỗ em ạ https://civ3-dev.framgia.vn/admin/repositories/framgia/fwplace/builds/27/violation

**Comment by nguyentien98** [comment]
> Vẫn còn lỗi convetion vài chỗ em ạ https://civ3-dev.framgia.vn/admin/repositories/framgia/fwplace/builds/27/violation

Em vừa sửa rồi a ạ. Thank anh :+1:

File: app/Repositories/WorkspaceRepository.php [hunk 21]
```diff
@@ -83,9 +111,9 @@ public function pluckWorkspace()
         return $this->model->pluck('name', 'id');
     }
 
-    public function getOneDate($workspace_id, $date, $filter = [])
+    public function getOneDate($workspaceId, $date, $filter = [])
     {
-        $workspace = $this->model->findOrFail($workspace_id);
+        $workspace = $this->model->findOrFail($workspaceId);
         if (!$workspace || !$date) {
             return null;
```
Comment (framgiaciv3) [inline app/Repositories/WorkspaceRepository.php:118]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/WorkspaceRepository.php</br>Whitespace found at end of line

File: app/Repositories/WorkspaceRepository.php [hunk 22]
```diff
@@ -83,9 +111,9 @@ public function pluckWorkspace()
...
         }
```
Comment (framgiaciv3) [inline app/Repositories/WorkspaceRepository.php:119]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/WorkspaceRepository.php</br>Whitespace found at end of line

File: app/Repositories/LocationRepository.php [hunk 23]
```diff
@@ -35,7 +35,11 @@ public function listLocation($filter)
 
     public function getByWorkspace($workspaceId)
     {
-        return $this->model->where('workspace_id', $workspaceId)->pluck('name', 'id')->prepend(__('--Chose Location--'), config('site.default_location'))->toArray();
+        return $this->model
+            ->where('workspace_id', $workspaceId)
+            ->pluck('name', 'id')
+            ->prepend(__('--Chose Location--'), config('site.default_location'))
```
Comment (BunnyPi04) [inline app/Repositories/LocationRepository.php]: sửa đây nhá

File: app/Repositories/ProgramRepository.php [hunk 24]
```diff
@@ -24,5 +25,4 @@ public function listProgram()
     {
         return $this->model->pluck('name', 'id')->prepend(__('<< Program >>'), '')->toArray();
```
Comment (BunnyPi04) [inline app/Repositories/ProgramRepository.php]: đây nữa

File: app/Repositories/WorkspaceRepository.php [hunk 25]
```diff
@@ -63,13 +87,17 @@ public function getByShift($workspace, $filter, $title, $shift, $color = null)
...
+            ->select(DB::raw('COUNT(user_id) as total, date as start, shift, CONCAT("'
+                    . $title
+                    . '", COUNT(user_id)) as title, ' . $color . 'as className'))
             ->whereBetween('date', [$filter['start'], $filter['end']])
             ->where('shift', $shift);
 
         if (array_key_exists('program_id', $filter) && $filter['program_id']) {
-            $get_user_by_program = DB::table('users')->where('program_id', $filter['program_id'])->pluck('id');
-            $shiftData = $shiftData->whereIn('user_id', $get_user_by_program);
+            $getUserByProgram = DB::table('users')
```
Comment (BunnyPi04) [inline app/Repositories/WorkspaceRepository.php]: sao ko dùng Model?

**Pull request merged**